grails-app/services/org/pih/warehouse/requisition/RequisitionService.groovy

```diff
@@ -837,6 +838,14 @@ class RequisitionService {
         }
     }
 
+    // TODO in Grails 3 move this method to @Service RequisitionDataService
```

Comment:
@drodzewicz I see here is a `TODO in Grails 3`. Is it something that will be fairly quick? Could you add it to this PR?

Reply:
Yes, this should be rather quick.
It requires for me to create a `RequisitionDataService` which would implement a `getRequisitionWithEvents` with `@Join("events")`